src/app/dashboard/help/page.jsx

```
@@ -0,0 +1,223 @@
+                            : "justify-start"
+                        } mb-2`}
+                      >
+                        <div
+                          className={`p-2 rounded-md max-w-xs ${
+                            message.role === "user"
+                              ? "bg-teal-500 text-white"
+                              : "bg-gray-100 text-black"
+                          }`}
+                        >
+                          <ReactMarkdown
+                            children={message.content}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)